docs/book/v3/strategies/serializable.md

```diff
@@ -0,0 +1,158 @@
+# Serializable
+
+The `SerializableStrategy` can be used for **serializing and deserializing PHP
+types to and from different representations**.
+
+The strategy uses [laminas-serializer](https://docs.laminas.dev/laminas-serializer/)
+for serializing and deserializing of data.
+
+## Basic usage
+
+The following code example shows standalone usage without adding the strategy
+to a hydrator.
+
+### Create and configure strategy
+
+Create the strategy and set a serializer adapter.
+
+```php
+$strategy = new Laminas\Hydrator\Strategy\SerializableStrategy(
+    new Laminas\Serializer\Adapter\Json()
+);
+```
+
+For available serializer adapters see the [documentation of laminas-serializer](https://docs.laminas.dev/laminas-serializer/adapter/).
+
+### Hydrate data
+
+```php
+$json = '[
+  {
+    "title": "Modern Love",
+    "duration": "4:46"
+  },
+  {
+    "title": "China Girl",
+    "duration": "5:32"
+  }
+]';
```

Comment:
I think it will be better if we have here json as string in one line, as in other places.
This is the only place I was a bit confused first, and then I've noted `'` before `[` ;-) 
```suggestion
$json = '[{"title":"Modern Love","duration":"4:46"},{"title":"China Girl","duration":"5:32"}]';
```